src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectSettingsPanel.java

```diff
@@ -25,6 +25,8 @@
 import java.awt.BorderLayout;
 import java.util.List;
 import javax.swing.JPanel;
+import javax.swing.event.ChangeEvent;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.swing.event.ChangeEvent'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)